### DISCUSSION TITLE
Offer only available options for printing: check format and language(s)

### DISCUSSION THREAD

**Discussion by kaare** [ledgersmb/LedgerSMB#5846] [Design]
1.5.7

Trying to output an order to csv returns w/ no response. Turns out lsmb tries to fetch a template 

SELECT * FROM "public"."template__get"('sales_order', NULL, 'csv')

which returns nothing.

csv option shouldn't be there if there's no way it'll work. 

Any other way to download the order as csv? It's handy in some situations

**Comment by ehuelsmann** [comment]
See also #811 .

**Comment by ehuelsmann** [comment]
The solution to be developed should offer the available options by intersecting available formats with the available templates for said formats as well as the available languages (which can be calculated as "on-disk languages"(for the generic template)+"specific language templates").
